src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -133,7 +134,7 @@ class DiskImageEditor extends Component {
 
     this.setState({
       showModal: true,
-      storageDomainSelectList: storageDomainsToSelectList(this.props.storageDomainList),
+      storageDomainSelectList: storageDomainsToSelectList(this.props.storageDomainList.filter(storageDomain => storageDomain.get('canUserUseDomain'))),
```

Comment:
You can drop the extra filter here since you added it to the `storageDomainsToSelectList` function.

Reply:
Oops... I forgot to drop it.